### PR TITLE
Update typography docs

### LINKED ISSUE #310: Bug - Typography
Current staging typography page annotates both subdued text and body copy as #424242 value. I believe the correct value should be as follows:

body: #333333 (gray 950)
subdued: #6B6B6B (gray 750) according to sketch. However this is not a value in our gray scale color palette. Might look at making this either #BDBDBD (gray 500) or #9E9E9E (Gray 600)

### PR DESCRIPTION
**Pull request by technabors**
Change notes: 

- The newest version of the typography doc was used to update the typography page. 
- New images created and exported as SVG / @2x.png. (PNG used). 
- Small content edits based on the content of the docs.
- closes #310 

Worth noting that the old way of presenting the typography was via css rather than images. This PR does not remove any CSS that was associated with this method. Let me know if this is an issue an I'll look into removing it or we can file and issue and get to it later.

### CODE REVIEW

File: _docs/style/typography.md [hunk 1]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
 
-### Sub-body text
-
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-Use sub-body text sparingly, since it is the least readable size. You can use it when smaller text is appropriate, such as for icon labels.
+### Caption
+
+Captions apply to images and visualizations. They should be used to clarify images and not introduce new, pertinent information.
```
Comment (nimbinatus) [inline _docs/style/typography.md]: Consider rewording the last sentence:
```markdown
Use the caption style to clarify images, and do **not** introduce new, pertinent information with a caption.
```

File: _docs/style/typography.md [hunk 2]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
 {% endcolumn %}
 </div>
 
-### Caption
+<div class="hxRow" markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Sub-body text
+
+Use sub-body text sparingly, since it is the least readable size. For example use this typography style for the following:
```
Comment (nimbinatus) [inline _docs/style/typography.md]: remove comma in the first sentence

Comment (nimbinatus) [inline _docs/style/typography.md]: s/`For example use this typography style for the following:`/`For example, use this typography style for the following contexts:`

File: _docs/style/typography.md [hunk 3]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
+{% endcolumn %}
+</div>
 
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-Captions apply to images and visualizations. They should be used to clarify images and not introduce new, pertinent information.
+### Subdued style
+
+Use the subdued style to allow more important information stand out. For example:
```
Comment (nimbinatus) [inline _docs/style/typography.md]: s/`information stand`/`information to stand`

Comment (nimbinatus) [inline _docs/style/typography.md]: Consider replacing the list as follows:
```markdown
For example, use the subdued style for keys in a key-value list to provide more visual prominence to the value or for non-critical information in lists, such as abbreviated dates or as a second line of text in a cell. 
```

Note the spelling fix for `prominence` here.

File: _docs/style/typography.md [hunk 4]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
+
+- Use the subdued style for keys in key-value lists to provide more visual prominenance to the value.
+- Use this style for non-critical information in lists, such as abbreviated dates, or as a second line of text in a cell
+
+Parameters for use:
```
Comment (nimbinatus) [inline _docs/style/typography.md]: s/`Parameters for use:`/`Note the following parameters for the use of subdued text:`

File: _docs/style/typography.md [hunk 5]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
+### Subdued style
+
+Use the subdued style to allow more important information stand out. For example:
+
+- Use the subdued style for keys in key-value lists to provide more visual prominenance to the value.
+- Use this style for non-critical information in lists, such as abbreviated dates, or as a second line of text in a cell
+
+Parameters for use: 
+
+- This style only applies Gray 750 color to the selected text, and does not affect font size or weight
```
Comment (nimbinatus) [inline _docs/style/typography.md]: s/`Gray 750`/`` `gray750` ``

Comment (nimbinatus) [inline _docs/style/typography.md]: Remove the comma, and add a period at the end.

File: _docs/style/typography.md [hunk 6]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
+- This is not the same as inactive style for links or buttons
```
Comment (nimbinatus) [inline _docs/style/typography.md]: period at the end of all of these

Comment (nimbinatus) [inline _docs/style/typography.md]: s/`This is`/`This style is`

File: _docs/style/typography.md [hunk 7]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
+- Use this style with text 14px or taller 
+- Do not subdue text on backgrounds darker than Gray 25 (#fafafa) or it will not pass accessibiity standards
```
Comment (nimbinatus) [inline _docs/style/typography.md]: Adjust as follows:
```markdown
Do **not**subdue text on backgrounds darker than `gray25` (`#fafafa`), or the text will not pass accessibility standards.
```
(Note the spelling fix above with `accessibility`.)

File: _docs/style/typography.md [hunk 8]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
-### Link
+## Lists
 
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-Links support styles for hover, active, and disabled states, and inherit size and spacing styles from the text to which they are applied. Helix does not support the visited state.
+### Ordered lists
+
+Ordered lists are usually reserved for instances in which the items must occur in a specific order, such as steps in a procedure, or when keeping count is important, such as a top 10 list.
```
Comment (nimbinatus) [inline _docs/style/typography.md]:
```markdown
Reserve ordered lists for instances...
```

File: _docs/style/typography.md [hunk 9]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
+
+If the order or the final count of the list items does not matter, don’t number them. Doing so creates confusion.
+
+For more information on when and how to use lists, check out the [Nielsen Norman Group’s article](https://www.nngroup.com/articles/presenting-bulleted-lists/) on this topic.
```
Comment (nimbinatus) [inline _docs/style/typography.md]: How to use lists in general or how to use ordered lists?

**Comment by nimbinatus** [comment]
Oh, and yes, I'd remove the excess CSS and the template page that was being used to generate the typography styles (https://github.com/rackerlabs/design-system/blob/staging/templates/_includes/type-styles.html). I think actually the current version just uses the helix-ui code, and I just didn't remove the specialized CSS from the original original as I wasn't sure if it was in use elsewhere...

File: _docs/style/typography.md [hunk 10]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
 ## Header styles
 
-### Page title
-
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
+### Page title
+
 Page titles are the highest level of hierarchy. There should only ever be one on any page.
```
[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: updated.

File: _docs/style/typography.md [hunk 11]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
 {% endcolumn %}
 </div>
 
-### Sub-section title
-
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-Sub-section titles are pretty versatile. Consider using them to title things like fieldsets, tables, Key-Value lists, and so on.
+### Container title
```
[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: @araiford See Laura's comment about the renaming of Sub-section title to Container title.

File: _docs/style/typography.md [hunk 12]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
+
+Container Titles are versatile and can be used to title content that exists inside a distinct visual container. For example:
```
[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: updated.

[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: updated.

[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: Updated the sentence. Unclear on the second part of this comment. Is the concern that the wording in container title and container sub-title are to similar?

File: _docs/style/typography.md [hunk 13]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
+
+- Title for tab content
+- Cards, [modals]({{site.baseurl}}/components/modal.html), or [fieldsets]({{site.baseurl}}/components/text-fields.html)
```
[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: updated.

File: _docs/style/typography.md [hunk 14]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
 {% endcolumn %}
 </div>
 
-### Container title
+<div class="hxRow" markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Container sub-title
+
+Container Sub-Title is a flexible style, and can be used for grouping information that exists in a titled container. For example use this typography style for the following:
```
[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: Updated.

[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: updatd.

File: _docs/style/typography.md [hunk 15]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
+
+- The labeling of sub-sections within tab content.
```
[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: removed.

File: _docs/style/typography.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: updated.

File: _docs/style/typography.md [hunk 16]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
+{% endcolumn %}
+</div>
 
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-Container titles are good for titling within elements like cards, modals, and so on.
+### Context title
+
+Use this typgraphic element to provide additional contextual labeling. For example use this typography style for the following:
```
[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: updated.

File: _docs/style/typography.md [hunk 17]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
+
+Container Sub-Title is a flexible style, and can be used for grouping information that exists in a titled container. For example use this typography style for the following:
+
+- The labeling of sub-sections within tab content.  
+- Labeling other tertiary content within a page, such as a small [popover]({{site.baseurl}}/components/popovers.html).
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [The checkbox component] [class:"image bg-light border"] %}
```
[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: removed.

File: _docs/style/typography.md [hunk 18]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
+
 Body text is the default style and should be used across the interface for most information.
 
+Paragraphs inherit body text style by default, and have 20px of bottom spacing.
+
 {% endcolumn %}
 
 {% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
-{% include type-styles.html htmlexample='<p>Body Text</p>' typeface="Roboto" weight="Regular (400)" fontsize="16px / 1rem" csscode="Gray 900" hex="#424242" bottomspace="24px / 1rem" linespace="1.5" %}
+{% figure [The checkbox component] [class:"image bg-light border"] %}
```
[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: removed.

File: _docs/style/typography.md (hunk 8 above)
Comment (technabors) [reply _docs/style/typography.md]: updated.

File: _docs/style/typography.md (hunk 4 above)
Comment (technabors) [reply _docs/style/typography.md]: updated.

File: _docs/style/typography.md (hunk 5 above)
Comment (technabors) [reply _docs/style/typography.md]: updated.

File: _docs/style/typography.md (hunk 3 above)
Comment (technabors) [reply _docs/style/typography.md]: updated.

File: _docs/style/typography.md (hunk 2 above)
Comment (technabors) [reply _docs/style/typography.md]: updated.

Comment (technabors) [reply _docs/style/typography.md]: removed.

File: _docs/style/typography.md (hunk 1 above)
Comment (technabors) [reply _docs/style/typography.md]: updated.

File: _docs/style/typography.md [hunk 19]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
-### Subdued style
-
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-You can use the subdued style with any text 14px and taller, and is used to let more important text stand out. The subdued style is not the same as an inactive link or button.
+### Link
 
-*Do not subdue text on backgrounds darker than Gray 25 (#fafafa) or it will not pass accessibility standards.*
+Links support styles for hover, active, and disabled states, and inherit size and spacing styles from the text to which they are applied. Helix does not support the visited state.
```
[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: updated.

File: _docs/style/typography.md [hunk 20]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
+
+Parameters for use:
+
+- **Do not** use the context title on backgrounds darker than the standard Gray 25 (#fafafa) or it will not pass accessibiity standards.
 
 {% endcolumn %}
 
 {% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
-{% include type-styles.html htmlexample='<h4 class="hxContainerTitle">Container title</h4>' typeface="Roboto" weight="Medium (500)" fontsize="16px / 1rem" csscode="Gray 900" hex="#424242" linespace="1" bottomspace="24px / 1.5rem" %}
+{% figure [The checkbox component] [class:"image bg-light border"] %}
```
[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: removed.

File: _docs/style/typography.md [hunk 21]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
 
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-Container titles are good for titling within elements like cards, modals, and so on.
+### Context title
+
+Use this typgraphic element to provide additional contextual labeling. For example use this typography style for the following:   
+
+- Adding a contextual title of “Cloud Server” above the server name on a device detail page.
```
[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: removed.

File: _docs/style/typography.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: updated.

File: _docs/style/typography.md [hunk 22]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
-### Body text
-
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
+### Body text
+
 Body text is the default style and should be used across the interface for most information.
 
+Paragraphs inherit body text style by default, and have 20px of bottom spacing.
```
[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: Updated to 'Paragraphs inherit body text style by default and have 20px bottom margin.'

File: _docs/style/typography.md [hunk 23]
```diff
@@ -16,170 +16,245 @@ helix-ui-javascript: false
...
 ## Body styles
 
-### Body text
-
 <div class="hxRow" markdown="1">
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
+### Body text
+
 Body text is the default style and should be used across the interface for most information.
```
[the remark this answers is not in the document]
Comment (technabors) [reply _docs/style/typography.md]: updated.

File: _docs/style/typography.md (hunk 3 above)
Comment (technabors) [reply _docs/style/typography.md]: updated.

File: _docs/style/typography.md (hunk 5 above)
Comment (technabors) [reply _docs/style/typography.md]: updated.

File: _docs/style/typography.md (hunk 6 above)
Comment (technabors) [reply _docs/style/typography.md]: updated.

Comment (technabors) [reply _docs/style/typography.md]: updated.

File: _docs/style/typography.md (hunk 7 above)
Comment (technabors) [reply _docs/style/typography.md]: updated.

File: _docs/style/typography.md (hunk 9 above)
Comment (technabors) [reply _docs/style/typography.md]: I'm just going to remove this link. It doesn't seem necessary.

**Comment by technabors** [comment]
As per discussion today we are holding this PR until the Helix-ui team has a chance to update typography. We will fix on both sides at once.

File: _docs/style/typography.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (araiford) [reply _docs/style/typography.md]: @nimbinatus 

See the typography section of the [bowling doc for October 17th](https://docs.google.com/document/d/13rhLD3z_IeIMV0N6LzWCnN5610iKPf3oAqvQLlegqjw/edit#heading=h.ww9qsd2i60ar).  We decided to change the names of our header hierarchy.  We do have corresponding jira cards in the SURF board for updating the class names in helix-ui.

[the remark this answers is not in the document]
Comment (nimbinatus) [reply _docs/style/typography.md]: :+1:

**Comment by technabors** [comment]
Closing this PR as it has become stale, never merged in, and the designs have shifted during this time period.

**Pull request closed without merging**